script/build_proxy_release.yaml

```diff
@@ -1,9 +0,0 @@
-# Used for automated builds triggered on Git tags pushed matching the regex:
```

Comment:
Earlier this week I changed the triggers to "0\.2\.\d+" since there might still be a need to do patch releases on that branch and mono-repo by itself has likely already broken the release so the triggers would work on master for 0.3 and later.

Reply:
The triggers have been updated to match 0.2 tags only, however if you delete this YAML file then the triggers won't work because the config file is missing.

Reply:
The last few 0.2 releases, and any future 0.2 releases, should come from the release-0.2 branch.  I could be mistaken, but I'd expect that Cloud Builder will look on this branch, rather than master, for the yaml file.  Either way I'd expect the build to fail to launch because the yaml file is still using N1_STANDARD_8 and as of last week I think CB now refuses to accept the deprecated x_STANDARD_y in factor of x_HIGHCPU_y.

Reply:
the file is deleted in master, the 0.2 tags will only apply to the release-0.2 branch so this should be safe.

Reply:
Gotcha, I didn't realize that 0.2 had been branched.  That alleviates my concerns about deleting the 0.2 release scripts.